packages/serve/src/index.ts

```diff
@@ -15,7 +15,11 @@ export default function serve(...args: string[]): void {
 
     const { webpackArgs, devServerArgs } = parseArgs(cli, args);
 
-    cli.getCompiler(webpackArgs, core).then((compiler): void => {
-        startDevServer(compiler, devServerArgs);
-    });
+    cli.getCompiler(webpackArgs, core)
+        .then((compiler): void => {
+            startDevServer(compiler, devServerArgs);
+        })
+        .catch((err: object): void => {
+            console.error(err);
```

Comment:
We should `throw` error, not logging

Reply:
So should I just throw an error, or log the err object and then throw the error?

Reply:
Just throw with error message